tests/network/curl_client.pm

```diff
@@ -0,0 +1,44 @@
+# SUSE"s openQA tests
+# #
+# # Copyright © 2019 SUSE LLC
+# #
+# # Copying and distribution of this file, with or without modification,
+# # are permitted in any medium without royalty provided the copyright
+# # notice and this notice are preserved.  This file is offered as-is,
+# # without any warranty.
+#
+# Summary: Test regression to curl.
+#  this test curl with http, https, ldap, ftp and ntlm auth
+#  poo#51536
+# Maintainer: Marcelo Martins <mmartins@suse.cz>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use mm_tests;
+use lockapi qw(mutex_create mutex_wait);
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
```

Comment:
You have to choose one of them, `$self->select_serial_terminal` or `select_console 'root-console';`

Reply:
`$self->select_serial_terminal` should select `root-console` if no serial console is available, so better to use `$self->select_serial_terminal`, but just my own point of view :-)

Reply:
done, removed root-console.